code/modules/mob/living/simple_mob/subtypes/animal/space/bats.dm

```diff
@@ -57,3 +57,20 @@
 	maxHealth = 60
 	health = 60
 	melee_damage_upper = 10
+
+/datum/category_item/catalogue/fauna/razor_bat
+	name = "Nispean Fauna - Razor Bat"
+	desc = "These sharp-toothed flying mammalian animals are adapted to the oxygen-deficient \
```

Comment:
I wouldn't say Nisp is especially oxygen deficient... although the percentage of oxygen is 10.5%, the pressure is 2.5atm, so the partial pressure of oxygen is actually a little bit higher than terrestrial air.